### PR TITLE
Article: Materialize

### PR DESCRIPTION
**Pull request by theflametrooper**
Refer #707

### CODE REVIEW

File: Materialize.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# Materialize
+
+Materialize is a Responsive CSS framework based on Google's [Material Design](https://design.google.com/spec/) Language.
+
+### Getting Started
```
Comment (abhisekp) [inline Materialize.md]: Use header 2

```md
- ### Getting Started
+ ## Getting Started
```

**Comment by abhisekp** [comment]
@theflametrooper would be nice to add a mini tutorial for materialize.

File: Materialize.md (hunk 1 above)
Comment (theflametrooper) [inline Materialize.md]: @abhisekp Is this for all subheadings?

Comment (Rafase282) [inline Materialize.md]: @theflametrooper normally you would go in ascending order, Heading 1, 2,3 .. n

The bigger the number the deeper it is into the root header. An article should ideally shoudl have One head 1, and the rest top headings would be heading 2, with subheadings as needed.

**Comment by Rafase282** [comment]
Mini tutorial? A link to their docs would be enough in my opinion. I think these articles should be more about letting campers know about something other than pure css and boostrap. Later we might add info from their docs but I don't see how you would make a mini guide for frameworks.

**Comment by abhisekp** [comment]
@theflametrooper @Rafase282 just adding a few relevant and important parts of the framework show casing the usefulness of the framework over pure CSS would be really beneficial. 
This will make the reader get attracted towards materialize (and CSS frameworks overall) and also let them get started with a very basic use case. 

_It really helps to get a kick start._ :boot: :8ball:

**Comment by theflametrooper** [comment]
@abhisekp @Rafase282 I will add this in a later PR. For now, let's get this merged!

**Comment by theflametrooper** [comment]
@alayek When you merge, please squash commits as I did most of this in browser ;)

File: Materialize.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Materialize
+
+Materialize is a Responsive CSS framework based on Google's [Material Design](https://design.google.com/spec/) Language.
+
+## Getting Started
+
+Here is a simple HTML template which includes the latest compiled and minified CSS and Javascript for the Materialize library.
+We have used a CDN in this example, but you can checkout other ways of installing Materialize [here](http://materializecss.com/getting-started).
```
Comment (alayek) [inline Materialize.md]: The line above can be below this, just above the example.

**Comment by alayek** [comment]
# LGTM :+1: 

@theflametrooper please update as mentioned in line note.

**Pull request merged**